### PR TITLE
make releasing manual

### PR DESCRIPTION
**Pull request by Wyvest**
## Description
Pushing to master now pushes to snapshots by default, and releasing requires manually triggering the publish workflow.

## Checklist
- [x] I made a clear description of what was changed
- [x] I stated why these changes were necessary
- [x] I updated documentation or said what needs to be updated
- [x] I made sure these changes are backwards compatible
- [x] This pull request is for one feature/bug fix

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -4,9 +4,6 @@ name: Release
 
 on:
   workflow_dispatch:
-  push:
-    branches:
```
Comment (pauliesnug) [inline .github/workflows/release.yml:8]: no

Comment (pauliesnug) [reply .github/workflows/release.yml:8]: bc it gets rid of the whole point of having branch protection

Comment (xtrm-en) [reply .github/workflows/release.yml:8]: no it doesn't?

File: .github/workflows/release-snapshots.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Release Workflow
+
+name: Release Snapshot
+
+on:
+  workflow_dispatch:
```
Comment (xtrm-en) [inline .github/workflows/release-snapshots.yml:6]: remove this

**Review by xtrm-en — DISMISSED** [review]
lgtm

**Review by pauliesnug — DISMISSED** [review]
dsadsadsadad

**Review by xtrm-en — APPROVED** [review]
pog

**Review by pauliesnug — APPROVED** [review]
h

**Pull request merged**